@navikt/core/css/darkside/loader.darkside.css

```diff
@@ -5,88 +5,101 @@
   animation: loader-rotate 1.8s linear infinite;
   stroke-width: 6;
 
-  --ac-loader-background-stroke-width: 5.9;
-}
+  --__axc-loader-stroke-width: 5.9;
 
-.navds-loader__foreground {
-  animation: loader-dasharray 1.8s ease-in-out infinite;
-  stroke-dasharray: 80px, 200px;
-  stroke-dashoffset: 0;
-  stroke: var(--ac-loader-stroke, var(--a-border-default));
-}
+  &.navds-loader--3xlarge {
```

Comment:
Why nest this?

Reply:
Personally like to have changes in local tokens before it is used ` --__axc-loader-stroke-width`. Feels disorienting to have "definition -> use -> definition -> definition" in some cases. But for a short file like this it doesn't really matter

Reply:
I think you misunderstand. I'm wondering why you put `&.navds-loader--3xlarge` etc. inside `.navds-loader {`?
Do you need the selector to be `.navds-loader.navds-loader--3xlarge`?